### ISSUE TITLE
Meeting 5/30 to Review AMD Requested Updates with 311

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#12417]
### Objective 
Walk through the requested enhancements following the requirements gathering meeting last year with Arterial Management and Signs & Markings.

### Participants
Carrie (311), Patrick (311), Christina, and Diana

**Comment by ChristinaTremel** [comment]
# Meeting Notes

- Action Item: Carrie will share spreadsheet with Activities and Future Activities

### ATD - DIG TESS Request

- This SR type is only used by 811
- Changes have been made to the intakes
- Resolution changes have not been made
    - Resolutions are sending out paging activities depending on intake

### ATD - School Zone Beacon Maintenance

- Changes to the intake have been completed
- Resolution changes have not been made
- Mobile portion of the service requests changes have not been made

### ATD - Signal Timing Questionnaire

- Seasonal 311 type and is only used as needed
- Does not go into the Knack interface

### ATD - Traffic Signal Maintenance

- Changes to the intake have not been completed
- SR is being renamed and that has not been completed yet
- This is the test SR. We want all the interface changes to be made before they start changing these things. Example adding All bulbs out
- Flex notes are changes - this shouldn’t be an issue
    - One thing to check up on: flex note with the “Time” type, will this be an issue? We don’t think so but it’ll be interesting to see what it looks like
- Blue text is that there are still questions they need to follow up with the SPOCS on
    - At what time are you noticing the skipping cycle occurring? - question is it character or time
- Power Outage (All Bulbs Out) is new SR type (this is what BC checked in with DTS on)

### Traffic Signal - New/Modification

- Brian C wants to split this SR type up
    - Traffic Signal - New
    - Traffic Signal - Modification
        - Definitely need to check in with John about this proposed change
- Changes to the intake have been completed, but have not been separated out between new and modification
- Changes to the Resolution have not been made
- Need to be split before they can make changes to the mobile app (currently doesn’t exist in the mobile app)
- Patrick thinks we need to make the changes on DTS side first before 311 splits these into 2 SRs

### SMD Knack App

- Check filters on Signs & Markings Data Tracker App for SR types to fix them since the SRs were renamed

**Comment by ChristinaTremel** [comment]
@johnclary - we will sync with you when you get back, but this was a more in depth meeting about the changes that AMD requested with 311 last year. 

311 will be scheduling meetings with AMD and SMD to check in on these enhancements since Carrie is the new 311 point of contact. She will include Diana and me on those future meetings.

**Comment by dianamartin** [comment]
# Diana's Notes 
Participants: 

- Patrick
- Carrie
- Christina
- Diana

## ATD - Dig Tess Request

- AMD SR - Yes
- Paging activity

## ATD - Questionnaire

- not bringing into Knack (seasonal)

## ATD - Signal Traffic Maintenance

- Paging activity
- Some new flex notes, as dates —> will come in as short text

## ATD - Signal Traffic New Modification

- AMD, are planning to split this up
- Waiting on new interface to do this modification

## ATD - Signal Traffic New (new)

- AMD, are planning to split this up
- Waiting on new interface to do this modification
    - going to 311 Mobile in the future

## ATD - Signal Traffic Modification (new)

- AMD, are planning to split this up
- Waiting on new interface to do this modification
    - going to 311 Mobile in the future

**Comment by johnclary** [comment]
👍 

we will need Robert to help with any new service request types in order to get them added to the integration. there should not be any code changes needed on our side. y'all will want to double-check that we're not using the SR type codes for any special knack logic: i don't _think_ that we are.

**Comment by dianamartin** [comment]
@johnclary I don't think we are either besides filter menus
